index.bs

```diff
@@ -743,11 +743,11 @@ When the <dfn method for="XRFrame">getViewerPose(|referenceSpace|)</dfn> method
 
 <div class="algorithm unstable" data-algorithm="get-pose">
 
-When the <dfn method for="XRFrame">getPose(|sourceSpace|, |destinationSpace|)</dfn> method is invoked, the user agent MUST run the following steps:
+When the <dfn method for="XRFrame">getPose(|space|, |referenceSpace|)</dfn> method is invoked, the user agent MUST run the following steps:
```

Comment:
I'm slightly concerned that using "referenceSpace" here may erroneously imply that the input can ONLY be an `XRReferenceSpace`. That said, naming is hard and there's not a lot a great options that I can think of here that address what you and Klaus were talking about. So maybe it's a non-issue?

Reply:
Yeah, that's fair. I think ultimately it's clear from the types, though.

_shrug_

Reply:
Hey, I jumped the gun on this and merged it before Nell had a chance to sign off, and she shares my concern about using `referenceSpace` as a name here. As a result I'm going to do a follow up to adjust the name again. I'll post a PR when I have something so we can make sure the name is agreeable all around.

Reply:
Sounds good. You can just revert https://github.com/immersive-web/webxr/pull/654/commits/a8563000d2857d9912cc574cafbb79d4b1e0cc3b